aws_doc_sdk_examples_tools/doc_gen.py

```diff
@@ -265,6 +266,14 @@ def validate(self):
             self.root,
         )
 
+    def validate_links(self):
+        for example in self.examples.values():
+            for language in example.languages.values():
+                for version in language.versions:
+                    if version.github:
+                        if not (self.root / version.github).exists():
+                            self.errors.append()
+
```

Comment:
Did you mean to delete this? It looks like it's never called.
If you do keep it you probably want to append an actual error.

Reply:
I did, this was a first attempt before deciding to do it the other way.